internal/storage/local/local.go

```diff
@@ -0,0 +1,130 @@
+package local
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+
+	strg "github.com/offen/docker-volume-backup/internal/storage"
+	t "github.com/offen/docker-volume-backup/internal/types"
+	u "github.com/offen/docker-volume-backup/internal/utilities"
+	"github.com/sirupsen/logrus"
+)
+
+type LocalStorage struct {
+	*strg.StorageBackend
+}
+
+// Specific init procedure for the local storage provider.
+func InitLocal(c *t.Config, l *logrus.Logger, s *t.Stats) *strg.StorageBackend {
+	a := &strg.StorageBackend{
+		Storage: &LocalStorage{},
+		Name:    "Local",
+		Logger:  l,
```

Comment:
What do you think about just passing in a logging function instead of the entire logger object. That way, the top level command can implement logging in any way it wants to without affecting the storages.

Reply:
You mean just passing a func (to the init method for example) which basically wraps the `Logger.Infof()`?